sys-apps/diskus/diskus-0.4.0.recipe

```diff
@@ -0,0 +1,256 @@
+CHECKSUM_SHA256_31="94258f53601af11e6a49f722422f6e3425c52b06245a5cf9bc09908b174f5e27"
+SOURCE_DIR_31="scopeguard-0.3.3"
+
+SOURCE_URI_32="https://static.crates.io/crates/semver/semver-0.9.0.crate"
+CHECKSUM_SHA256_32="1d7eb9ef2c18661902cc47e535f9bc51b78acd254da71d375c2f6720d9a40403"
+SOURCE_DIR_32="semver-0.9.0"
+
+SOURCE_URI_33="https://static.crates.io/crates/semver-parser/semver-parser-0.7.0.crate"
+CHECKSUM_SHA256_33="388a1df253eca08550bef6c72392cfe7c30914bf41df5269b68cbd6ff8f570a3"
+SOURCE_DIR_33="semver-parser-0.7.0"
+
+SOURCE_URI_34="https://static.crates.io/crates/smallvec/smallvec-0.6.5.crate"
+CHECKSUM_SHA256_34="153ffa32fd170e9944f7e0838edf824a754ec4c1fc64746fcc9fe1f8fa602e5d"
+SOURCE_DIR_34="smallvec-0.6.5"
+
+SOURCE_URI_35="https://static.crates.io/crates/stable_deref_trait/stable_deref_trait-1.1.1.crate"
+CHECKSUM_SHA256_35="dba1a27d3efae4351c8051072d619e3ade2820635c3958d826bfea39d59b54c8"
+SOURCE_DIR_35="stable_deref_trait-1.1.1"
+
+SOURCE_URI_36="https://static.crates.io/crates/strsim/strsim-0.7.0.crate"
+CHECKSUM_SHA256_36="bb4f380125926a99e52bc279241539c018323fab05ad6368b56f93d9369ff550"
+SOURCE_DIR_36="strsim-0.7.0"
+
+SOURCE_URI_37="https://static.crates.io/crates/term_size/term_size-0.3.1.crate"
+CHECKSUM_SHA256_37="9e5b9a66db815dcfd2da92db471106457082577c3c278d4138ab3e3b4e189327"
+SOURCE_DIR_37="term_size-0.3.1"
+PATCHES_37="$SOURCE_DIR_37.patch"
+
+SOURCE_URI_38="https://static.crates.io/crates/termion/termion-1.5.1.crate"
+CHECKSUM_SHA256_38="689a3bdfaab439fd92bc87df5c4c78417d3cbe537487274e9b0b2dce76e92096"
+SOURCE_DIR_38="termion-1.5.1"
+PATCHES_38="$SOURCE_DIR_38.patch"
+
+SOURCE_URI_39="https://static.crates.io/crates/textwrap/textwrap-0.10.0.crate"
+CHECKSUM_SHA256_39="307686869c93e71f94da64286f9a9524c0f308a9e1c87a583de8e9c9039ad3f6"
+SOURCE_DIR_39="textwrap-0.10.0"
+
+SOURCE_URI_40="https://static.crates.io/crates/unicode-width/unicode-width-0.1.5.crate"
+CHECKSUM_SHA256_40="882386231c45df4700b275c7ff55b6f3698780a650026380e72dabe76fa46526"
+SOURCE_DIR_40="unicode-width-0.1.5"
+
+SOURCE_URI_41="https://static.crates.io/crates/unreachable/unreachable-1.0.0.crate"
+CHECKSUM_SHA256_41="382810877fe448991dfc7f0dd6e3ae5d58088fd0ea5e35189655f84e6814fa56"
+SOURCE_DIR_41="unreachable-1.0.0"
+
+SOURCE_URI_42="https://static.crates.io/crates/vec_map/vec_map-0.8.1.crate"
+CHECKSUM_SHA256_42="05c78687fb1a80548ae3250346c3db86a80a7cdd77bda190189f2d0a0987c81a"
+SOURCE_DIR_42="vec_map-0.8.1"
+
+SOURCE_URI_43="https://static.crates.io/crates/version_check/version_check-0.1.5.crate"
+CHECKSUM_SHA256_43="914b1a6776c4c929a602fafd8bc742e06365d4bcbe48c30f9cca5824f70dc9dd"
+SOURCE_DIR_43="version_check-0.1.5"
+
+SOURCE_URI_44="https://static.crates.io/crates/void/void-1.0.2.crate"
+CHECKSUM_SHA256_44="6a02e4885ed3bc0f2de90ea6dd45ebcbb66dacffe03547fadbb0eeae2770887d"
+SOURCE_DIR_44="void-1.0.2"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
+fi
```

Comment:
```diff
 ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+commandBinDir=$binDir
 if [ "$targetArchitecture" = x86_gcc2 ]; then
 SECONDARY_ARCHITECTURES="x86"
+commandBinDir=$prefix/bin
 fi
```

To install without the `-x86` suffix on gcc2 secondary arch, you'd have to define `commandBinDir`.

@fbrosson, can you help with the aesthetic?

Reply:
It does look pretty weird without the indent, but I applied it for now since @fbrosson hasn't responded yet.